src/library/banshee/bansheefeature.cpp

```diff
@@ -79,7 +79,7 @@ void BansheeFeature::activate() {
             qDebug() << m_databaseFile << "does not exist";
         }
 
-        if (!m_connection.open(m_databaseFile)) {
+        if (!Sandbox::askForAccess(m_databaseFile) && !m_connection.open(m_databaseFile)) {
```

Comment:
I guess you intended to use `||` and not `&&`?

Reply:
Yes, thank you. This is why we do code review. :)